tests/models/idefics3/test_processing_idefics3.py

```diff
@@ -29,7 +29,7 @@
 
 @require_torch
 @require_vision
-class Idefics3ProcessorTest(unittest.TestCase):
+class Idefics3ProcessorTest(ProcessorTesterMixin):
```

Comment:
This should inherit from both `unittest.TestCase` and `ProcessorTesterMixin`

Reply:
oh no! multi inheritance! 💀
changed it to inherit from both!